core/storage/classifier/gae_models.py

```diff
@@ -95,19 +100,19 @@ def create(
                 data used by the classifier.
 
         Returns:
-            ID of the new ClassifierModel entry.
+            ID of the new ClassifierDataModel entry.
 
         Raises:
             Exception: A model with the same ID already exists.
         """
 
         instance_id = cls._generate_id(exp_id)
```

Comment:
Another final query, do I rename all occurrences of classifier_id in Domain layer with job_request_id. Because the classifier_id is the job_request_id (and I specify this property in gae_models).